RevenueCatUI/Templates/Components/Packages/PackageGroup/PackageGroupComponentView.swift

```diff
@@ -20,11 +20,16 @@ import SwiftUI
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 struct PackageGroupComponentView: View {
 
+    @EnvironmentObject
+    private var paywallState: PaywallState
+
     let viewModel: PackageGroupComponentViewModel
 
     var body: some View {
-        // WIP: Do something with default package id and selection
         StackComponentView(viewModel: self.viewModel.stackComponentViewModel)
+            .onAppear {
+                self.paywallState.select(package: self.viewModel.defaultPackage)
+            }
```

Comment:
Just checking, but can we trust `onAppear`? E.g. what happens if the user selects a non-default package, clicks a link (e.g. to read the privacy policy), and comes back? Is the non-default package still selected, or does `onAppear` fire again?

Reply:
The non-default should still be selected! `onAppear` only gets called one (its poorly named 😛)